webapp_server/chart/cosmotech-business-webapp/values.yaml

```diff
@@ -0,0 +1,30 @@
+# name is the name of the deployment, ingress and service that will be created
+name: cosmotech-business-webapp
+# imageName is the name of the docker image containing your webapp server (e.g. ghcr.io/cosmo-tech/azure-sample-webapp)
+imageName: ""
```

Comment:
Not sure what are the best practices here, but in the api (and the modeling api by copy), the image info are structured in sub-field `image.repository` (instead of name), `image.tag` (and `image.pullPolicy=Always` that is missing here)